frontend/apps/cloudstorage/webdavapi.lua

```diff
@@ -11,6 +11,13 @@ local logger = require("logger")
 local WebDavApi = {
 }
 
+function WebDavApi:jointPath( address, path )
```

Comment:
joined?

Reply:
Or, as it is a method and better be a verb: :joinPath() or :getJoinedPath() :)

Reply:
Done again